src/app/components/form-user-profile/form-user-profile.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import {FormControl, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-form-user-profile',
+  templateUrl: './form-user-profile.component.html',
+  styleUrls: ['./form-user-profile.component.less']
+})
+export class FormUserProfileComponent implements OnInit {
+  passwordChange = true;
+  hide = true;
+  email = new FormControl('', [Validators.required, Validators.email]);
```

Comment:
you need to create formGroup for form and add controls in it

Reply:
I don't understand what you mean

Reply:
It can help you [Angular Form](https://metanit.com/web/angular2/5.5.php)